sa3ed_app/api/CaseStatus.py

```diff
@@ -0,0 +1,17 @@
+import frappe
+
+@frappe.whitelist()
+def follow_up_case(case_id):
+    
+    lost_person = frappe.db.get_value('Lost Person', {'name': case_id}, ['case_status'], as_dict=True)
+    
+    if lost_person:
+        return {'case_status': lost_person['case_status']}
+    
+    
+    founded_person = frappe.db.get_value('Founded Person', {'name': case_id}, ['case_status'], as_dict=True)
+    
+    if founded_person:
+        return {'case_status': founded_person['case_status']}
```

Comment:
Hence we are building APIs not regular functions, you should consider implement a response json object like
```
response = {
"status_code": 200, // for success
"success": true,
"data": {
    "case_status": founded_person['case_status']
 }

return response
 
 
}
```